Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/AssetTests.cs

```diff
@@ -0,0 +1,277 @@
+﻿using Kentico.Kontent.Management.Models.Shared;
+using Kentico.Kontent.Management.Tests.Unit.Base;
+using Kentico.Kontent.Management.Tests.Unit.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+using FluentAssertions;
+using Kentico.Kontent.Management.Extenstions;
+using Kentico.Kontent.Management.Models.Assets;
+using Kentico.Kontent.Management.Models.StronglyTyped;
+
+namespace Kentico.Kontent.Management.Tests.Unit.ManagementClientTests
+{
+    public class AssetTests : IClassFixture<FileSystemFixture>
+    {
+        private readonly FileSystemFixture _fileSystemFixture;
+
+        public AssetTests(FileSystemFixture fileSystemFixture)
```

Comment:
Usually, when unit tests are created for a new entity all integration tests are re-implemented. I see this is not a case and I understand that it is out of the scope for this task but it would be great to keep track of which tests are missing. I will create an issue.

Reply:
Nulls - that means more than twice as many tests :D Ok, will add them although IMHO they feel unnecessary :D

Reply:
Added the tests for null checks and updated implementation accordingly.